contracts/utils/CHANGELOG.json

```diff
@@ -1,4 +1,17 @@
 [
+    {
+        "version": "2.0.5",
+        "changes": [
+            {
+                "note": "Fix bug in `LibBytes.slice` and `LibBytes.sliceDestructive`",
```

Comment:
Will this break any existing code in other contract packages?

Reply:
* nvm - doesn't look like `slice` / `sliceDestructive` are being used elsewhere.